readme.rst

```diff
@@ -50,6 +50,12 @@ Download the latest stable N98-Magerun phar-file from the file-server_:
 
    wget https://files.magerun.net/n98-magerun.phar
 
+or if you have problems with SSL certificate:
```

Comment:
Curl is not the answer for having unspecified problems with SSL certificates, however providing the syntax is helpful. I'll take this over, just my 2 cents.